build.gradle

```diff
@@ -102,6 +102,16 @@ task integrationTest(type: Test) {
     }
 }
 
+task integrationTestSignRequest(type: Test) {
```

Comment:
We can remove this and focus on running all IntegrationTests on the Travis later.